### ISSUE TITLE
Error when using Route::multilingual()->middleware()

### ISSUE THREAD

**Issue by vaskolukas** [chinleung/laravel-multilingual-routes#39]
**Describe the bug**

I have this route:

```php
Route::multilingual('/test', [TestController::class, 'index'])->middleware(['can:open something'])->name('test.index');
```

And I am receiving this error:

> Call to undefined method ChinLeung\MultilingualRoutes\MultilingualRoutePendingRegistration::middleware()

Without attached middleware routes working fine.

**Comment by chinleung** [comment]
Hi @vaskolukas,

I'm going to have a look at the middleware. For the moment, you can use this as a workaround:

```php
Route::middleware(['can:open something'])->group(function () {
    Route::multilingual('/test', [TestController::class, 'index'])->name('test.index');
});
```

**Comment by chinleung** [comment]
The middleware method is not available as of [v2.6.0](https://github.com/chinleung/laravel-multilingual-routes/releases/tag/v2.6.0).
